docs/dsl_resource.md

```diff
@@ -98,3 +98,23 @@ end
 ```
 
 For a full example, see our [example resource](https://github.com/chef/inspec/blob/master/examples/profile/libraries/example_config.rb).
+
+## Lazy Loading
+
+As of InSpec v4.16, resources are lazily loaded into the `inspec`
+process upon use. This greatly speeds up the initial startup costs of
+the `inspec` process and only loads what you need to use (eg, `inspec
+--version` no longer runs for 10 seconds!).
+
+It does change the way the resource registry is handled in ways that
```

Comment:
What is the "it" referring to in this sentence? I assumed it was "lazy loading," but in the context of the paragraph, I feel that I may have incorrectly assumed.

Reply:
Yes, I can change that to "Lazy loading"